assets/queries/ansible/aws/ec2_instance_using_default_security_group/query.rego

```diff
@@ -0,0 +1,34 @@
+package Cx
+
+import data.generic.ansible as ans_lib
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	task := ans_lib.tasks[id][t]
+	modules := {"amazon.aws.ec2", "ec2"}
+	ec2 := task[modules[m]]
+	ans_lib.checkState(ec2)
+
+	sgs := {"group", "group_id"}
+
+	sgName := get_name(ec2[sgs[s]])
+
+	contains(lower(sgName), "default")
+
+	result := {
+		"documentId": id,
+		"searchKey": sprintf("name={{%s}}.{{%s}}.%s", [task.name, modules[m], sgs[s]]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("'%s' is not using default security group", [sgs[s]]),
+		"keyActualValue":  sprintf("'%s' is using default security group", [sgs[s]]),
+		"searchLine": common_lib.build_search_line(["playbooks", t, modules[m], sgs[s]], []),
+	}
+}
+
+get_name(group) = name {
+	is_array(group)
+	name := group[g]
```

Comment:
I think this line should be changed, since can generate "multiples returns" and rego does not allow this